### PR TITLE
device: show address on device display

### LINKED ISSUE #2184: [TREZOR] Feature request: Display address on device
In Trezor's Bitcoin web wallet, I can click a button that displays the receiving address on Trezor's screen, confirming that it was actually created on my device. Also, I can display a QR code with the receiving address on Trezor's screen.

This function is missing on monero-gui.

@ph4r05

### PR DESCRIPTION
**Pull request by ph4r05**
- Fixes #2184 
- Requires https://github.com/monero-project/monero/pull/5594 to be merged to monero-core and monero submodule update

Screenshot 2019-06-17 at 19 39 45
Screenshot 2019-06-17 at 19 40 19
Screenshot 2019-06-17 at 19 40 34

### CODE REVIEW

**Comment by xmrdsc** [comment]
Nice one!

File: src/libwalletqt/Wallet.cpp [hunk 1]
```diff
@@ -347,6 +352,20 @@ void Wallet::setSubaddressLabel(quint32 accountIndex, quint32 addressIndex, cons
 {
     m_walletImpl->setSubaddressLabel(accountIndex, addressIndex, label.toStdString());
 }
+void Wallet::deviceShowAddressAsync(quint32 accountIndex, quint32 addressIndex, const QString &paymentId)
+{
+    QFuture<void> future = QtConcurrent::run(m_walletImpl, &Monero::Wallet::deviceShowAddress,
```
Comment (xiphon) [inline src/libwalletqt/Wallet.cpp]: Could you please update the code to invoke async calls through `FutureScheduler`.
Ex.:
https://github.com/monero-project/monero-gui/blob/master/src/libwalletqt/Wallet.cpp#L465 (signal)
or
https://github.com/monero-project/monero-gui/blob/master/src/libwalletqt/Wallet.cpp#L642 (js callback)

**Comment by ph4r05** [comment]
@xiphon I've adapted the async in the 13b6955 commit (rebased, force pushed). But github still shows the old commit even though the PR feed shows the force-push.

**Comment by ph4r05** [comment]
OK github shows it correctly after another force push.
Once required PRs are merged in the monero-core I will change the title and remove WIP.

**Comment by ph4r05** [comment]
Thanks @luigi1111  for merging https://github.com/monero-project/monero/pull/5594, this PR is ready to go after monero.git submodule is updated.

**Pull request merged**